RCPTT_Tests/ExperimentalDetails/AddEditRemoveAndClearUserDetails.test

```diff
@@ -0,0 +1,116 @@
+--- RCPTT testcase ---
+Format-Version: 1.0
+Element-Name: AddEditRemoveAndClearUserDetails
+Element-Type: testcase
+Element-Version: 3.0
+External-Reference: 
+Id: _G9wcgEUCEea3c7YIX-7B3A
+Runtime-Version: 2.0.1.201508250612
+Save-Time: 8/9/16 3:50 PM
+Testcase-Type: ecl
+
+------=_.content-0a7243a0-75d3-3d5f-9791-539de0e5b7ac
+Content-Type: text/ecl
+Entry-Name: .content
+
+switch_to_experiment_details_view
+
+/*
+*	Clear and assert dashboard empty
+*/
+get-view "My View" | get-button Clear | click
+get-view Dashboard | get-label "" -after [get-label "Users:"] | get-property "getText()" | equals "" | verify-true
+
+/*
+*	Add three default users and assert dashboard
+*/
+with [get-view "My View"] {
+    with [get-button Add] {
+        click
+        click
+        click
+        click
+    }
+}
+get-view Dashboard | get-label "DEFAULT_USER, DEFAULT_USER_1, DEFAULT_USER_2 and DEFAULT_USER_3" | get-property "getText()" 
```

Comment:
Multiple repeated string. Use a defined value and concatenate it with the index to avoid repetition